c7n/resources/acm.py

```diff
@@ -53,6 +53,9 @@ def augment(self, resources):
             super(DescribeCertificate, self).augment(resources))
 
 
+register_universal_tags(Certificate.filter_registry, Certificate.action_registry)
```

Comment:
legacy=False should be used btw else we get older aliases

Reply:
Are you talking about compatibility=False? Didn't see anything related to legacy in the actual function implementation. I can put that in as a new PR?